dbt/include/spark/macros/materializations/view.sql

```diff
@@ -1,25 +1,69 @@
-{% materialization view, adapter = 'spark' %}
-  
+{%- materialization view, adapter='spark' -%}
+
   {%- set identifier = model['alias'] -%}
+  {%- set tmp_identifier = model['name'] + '__dbt_tmp' -%}
```

Comment:
In the case of the view materialization, since Spark supports `create or replace view`, I think we should move in the direction of Snowflake, BigQuery, and Presto (https://github.com/fishtown-analytics/dbt-presto/pull/13) and away from the clever transactional approach in Postgres/Redshift (default).

I think this materialization would look like:
```sql
{% materialization view, adapter='spark' -%}
    {{ return(create_or_replace_view()) }}
{%- endmaterialization %}
```

And in the [adapter macro](dbt/include/spark/macros/adapters.sql):
```sql
{% macro spark__create_view_as(relation, sql) -%}
  create or replace view
    {{ relation }}
  as
    {{ sql }}
{% endmacro %}
```

This would slim the materialization code down _significantly_, and leverage some existing art in `dbt-core`'s `create_or_replace_view` macro.